csm-extensions/services/dev-postgres/Dockerfile-setup

```diff
@@ -0,0 +1,17 @@
+FROM cf-usb-sidecar-buildbase:latest
+
+COPY chart/scf-connector.sh /usr/local/bin/
+COPY chart/authorize_ca.sh  /usr/local/bin/
+WORKDIR /
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y unzip
```

Comment:
Should have an `apt-get clean` at the end of these commands.

Reply:
Fixed by multi-stage building (so we just throw away the whole container).